modules/social_features/social_post/modules/social_post_album/templates/field--post--type--image--album.html.twig

```diff
@@ -0,0 +1,16 @@
+{{ attach_library('socialbase/photoswipe.image') }}
+
+{# @see field--field-post-image.html.twig #}
+<div class=" asdasdasd" itemscope itemtype="http://schema.org/ImageGallery"
```

Comment:
Don't forget about this class 😂

Reply:
Sorry, but this is draft PR, and also, this one is not on the review yet. I know about that, thanks!

Reply:
Ha sorry about that, was funny to find though 😊  No worries!